Gemfile.lock

```diff
@@ -134,7 +134,7 @@ GEM
     cucumber-wire (0.0.1)
     database_cleaner (1.6.2)
     debug_inspector (0.0.3)
-    devise (4.4.0)
+    devise (4.4.1)
```

Comment:
@tansaku just making a review comment here to avoid merging it since travis has not run.